content/news.pml

```diff
@@ -37,6 +37,132 @@
 </div>
 -->
 
+<!----------------------------------------------------------->
+<div class="news-item">
+  24 May 2021: 4 concrete ways to support seL4 now
+</div>
+<div>
+  <p>
+    <img src="/images/sel4-foundation-logo.svg" style="width: 15%;
+    padding-left:10px; float:right" alt="RISC-V logo">
+  </p>
+  <p>
+    It has been heart-warming to see the response of the seL4
+    community to the news of the dismantlement of the Trustworthy
+    Systems group (TS), creator of seL4.
+  </p>
+
+  <p>
+    The seL4 Foundation, as well as its members and ecosystem, want to
+    reinforce their commitment to the success and support of seL4. The
+    TS team will be rebuilding at UNSW and a number of
+    Foundation-endorsed <a href="/Foundation/Services">seL4 services
+    providers</a> as well as the newly created <a
+    href="https://proofcraft.systems">Proofcraft</a> verification
+    company are dedicated to support seL4 in the future.
+  </p>
+  <p>
+    With this recent news, it’s never been more important to work
+    together as a community to support seL4 and the transition of the
+    support and development to the Foundation and its members.
+  </p>
+  <p>
+    People have been asking how to help. Offers of support keep
+    coming. While we keep working on further long-term support and
+    projects, there are a few things that are needed right now. <a
+    href="mailto:foundation@sel4.systems">Contact us</a> if you want
+    to discuss.
+  </p>
+  <p>
+    Here are 4 concrete things that can be done now. Spread the word
+    to people who care about seL4’s future:
+  </p>
+  <ol>
+    <li>
+
+      <b> donations and crowdfunding, in particular towards the
+	  “continuity project".</b></br>
+
+      <a
+      href="/Foundation/Support">https://sel4.systems/Foundation/Support/</a></br>
+      <a
+      href="https://crowdfunding.lfx.linuxfoundation.org/projects/e94c998c-bac2-4224-b0ae-23f265fdd1a5">https://crowdfunding.lfx.linuxfoundation.org/projects/e94c998c-bac2-4224-b0ae-23f265fdd1a5</a></br>
```

Comment:
Might be nicer to use a name for the link instead of the long code, maybe

```suggestion
      href="https://crowdfunding.lfx.linuxfoundation.org/projects/e94c998c-bac2-4224-b0ae-23f265fdd1a5">seL4 crowdfunding page</a></br>
```